src/code/game.c

```diff
@@ -16,22 +16,22 @@ void Game_UpdateFramerateVariables(s32 divisor) {
     gFramerateDivisorThird = (f32) (divisor / 3.0f);
 }
 
-void Game_SetFramerateDivisor(GameState *gameState, s32 divisor) {
+void Game_SetFramerateDivisor(GameState* this, s32 divisor) {
     R_UPDATE_RATE = (s16) divisor;
-    gameState->framerateDivisor = divisor;
+    this->framerateDivisor = divisor;
     Game_UpdateFramerateVariables(divisor);
 }
 
 void GameState_SetFBFilter(Gfx** gfx, u32 arg1) {
-    Gfx* _gfx = *gfx;
+    Gfx* gfxP = *gfx;
 
```

Comment:
Don't change or ptrGfx

Reply:
@Rozelette I kinda wanted more input on this because I really don't like the use of a prefixed _ in names of things. iirc, it also has issues with some compilers (i think we had this problem when dealing with object stuff in oot).

Reply:
This only applies to names with 2 underscores or 1 underscore followed by a uppercase letter

Reply:
(I dont like to use a shortform for the ptr since gfx is already a shortform.)

Reply:
`gfxPtr` would be my ideal change here.

Reply:
`dlist` is used in this kind of stuff in other places, so you should consider it too.